src/Cli/dotnet/CommonOptionsExtension.cs

```diff
@@ -38,9 +34,63 @@ public static LoggerVerbosity ToLoggerVerbosity(this VerbosityOptions verbosityO
                     verbosity = Build.Framework.LoggerVerbosity.Quiet;
                     break;
             }
+            return verbosity;
+        }
 
+        public static bool IsDetailedOrDiagnostic(this VerbosityOptions verbosity)
+        {
+            return verbosity.Equals(VerbosityOptions.diag) ||
+                verbosity.Equals(VerbosityOptions.diagnostic) ||
+                verbosity.Equals(VerbosityOptions.d) ||
+                verbosity.Equals(VerbosityOptions.detailed);
+        }
 
-            return verbosity;
+        public static bool IsQuiet(this VerbosityOptions verbosity)
+        {
+            return verbosity.Equals(VerbosityOptions.q) ||
+                verbosity.Equals(VerbosityOptions.quiet);
```

Comment:
Nit pick: I'd probably use the `ToLogLevel` here - seems bit more condesed

```
return verbosity.ToLogLevel == LogLevel.None;
```

Reply:
`VerbosityOptions` is not always converted to `LogLevel`, also it not 1-to-1 match. I would prefer this to stay as is.